soapui/src/main/java/com/eviware/soapui/tools/AbstractSoapUIRunner.java

```diff
@@ -76,11 +78,19 @@ protected void initGroovyLog() {
         if (!groovyLogInitialized) {
             Logger logger = Logger.getLogger("groovy.log");
 
-            ConsoleAppender appender = new ConsoleAppender();
-            appender.setWriter(new OutputStreamWriter(System.out));
-            appender.setLayout(new PatternLayout("%d{ABSOLUTE} %-5p [%c{1}] %m%n"));
-            logger.addAppender(appender);
-
+            // ensure there is a ConsoleAppender defined, adding one if necessary
+            boolean addAConsoleAppender = true;

```

Comment:
I'd extract lines 82-93 to it's own method, named something like ensureConsoleAppenderIsDefined(), and get rid of both this comment and the boolean variable - you could just return from the method